docs/metrics/README.md

```diff
@@ -417,6 +417,73 @@ SDK to reclaim unused metric points.
   Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#overflow-attribute)
   become stable, this feature will be turned on by default.
 
+#### Cardinality overflow example
+
+In the following example we set the maximum number of `MetricPoint`s allowed to
+be `3` for two metrics. This means that for these metrics, the SDK will only
+export measurements for up to `3` distinct key/value combinations.
+
+> [!NOTE]
+> One `MetricPoint` is reserved on every `Metric` for the special case where
+  there are no key/value pairs associated with a measurement. When choosing a
+  cardinality limit users should account for this special case.
+
+```csharp
+Counter<long> MyFruitCounter = MyMeter.CreateCounter<long>("MyFruitCounter");
+Counter<long> AnotherFruitCounter = MyMeter.CreateCounter<long>("AnotherFruitCounter");
+
+using var meterProvider = Sdk.CreateMeterProviderBuilder()
+    .AddMeter("*")
+    .AddConsoleExporter()
+    .AddView(i =>
+    {
+        if (i.Name == "MyFruitCounter" || i.Name == "AnotherFruitCounter")
```

Comment:
Now that we are offering per instrument cardinality configuration, a better example would be to just the set the limit for one instrument. That way we can show that the other instrument wasn't affected by the cardinality limit and exported all the MetricPoints.

Reply:
See: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5349#discussion_r1486723800

The `AnotherFruitCounter` example is kind of interesting because you only get 2 things when you ask for 3. Might be worth mentioning? For now I added a third counter to show the defaults apply to it.